__tests__/app.test.js

```diff
@@ -30,4 +31,27 @@ describe("app tests", () => {
           })
         })
     })
-})
\ No newline at end of file
+  })
+  describe("task 2 get reviews", () => {
+    test("status 200", () => {
+      return request(app).get("/api/reviews/").expect(200);
+    })
+    test("returns reviews including all properties", () => {
+      return request(app).get("/api/reviews/").expect(200).then((response) => {
+        const reviewObjs = response.body.reviews;
+        expect(reviewObjs).toHaveLength(13);
+        reviewObjs.forEach((obj) => {
+          expect.objectContaining({title: expect.any(String), designer: expect.any(String), owner: expect.any(String), review_img_url: expect.any(String), review_body: expect.any(String), category: expect.any(String), created_at: expect.any(String), votes: expect.any(Number)});
+          expect(obj).toHaveProperty("title");
+          expect(obj).toHaveProperty("designer");
+          expect(obj).toHaveProperty("owner");
+          expect(obj).toHaveProperty("review_img_url");
+          expect(obj).toHaveProperty("review_body");
+          expect(obj).toHaveProperty("category");
+          expect(obj).toHaveProperty("created_at");
+          expect(obj).toHaveProperty("votes");
+        })
```

Comment:
we've already tested above that the returned array contains an object containing all of these properties so `.toHaveProperty` is a little redundant here as it's already tested above